frontend/src/pages/modelServing/screens/metrics/bias/BiasMetricConfigSelector.tsx

```diff
@@ -26,71 +14,70 @@ const BiasMetricConfigSelector: React.FC<BiasMetricConfigSelectorProps> = ({
   initialSelections,
 }) => {
   const { biasMetricConfigs, loaded } = useModelBiasData();
-
-  const [isOpen, setIsOpen] = React.useState(false);
-
-  const selected = React.useMemo(
-    () => initialSelections.map(createBiasSelectOption),
-    [initialSelections],
-  );
-
+  const [uiSelections, setUISelections] = React.useState<string>();
+  const [currentSelections, setCurrentSelections] = React.useState<string>();
   const elementId = React.useId();
 
-  const changeState = React.useCallback(
-    (options: BiasSelectOption[]) => {
-      onChange(options.map((x) => x.biasMetricConfig));
-    },
-    [onChange],
-  );
+  const spdConfigs = biasMetricConfigs.filter((x) => x.metricType === BiasMetricType.SPD);
+  const dirConfigs = biasMetricConfigs.filter((x) => x.metricType === BiasMetricType.DIR);
 
   return (
     <div style={{ maxWidth: '600px' }}>
       <span id={elementId} hidden>
         Select the metrics to display charts for
       </span>
-      <Select
-        variant={SelectVariant.typeaheadMulti}
-        typeAheadAriaLabel="Select a metric"
-        onToggle={(e, val) => setIsOpen(val)}
-        onSelect={(event, item) => {
-          if (isBiasSelectOption(item)) {
-            if (selected.find(byId(item))) {
-              // User has de-selected an item.
-              changeState(selected.filter(byNotId(item)));
-            } else {
-              // User has selected an item.
-              changeState([...selected, item]);
+      <div>ALL: {biasMetricConfigs.map((c) => c.name).join(', ')}</div>
+      <div>UI: {uiSelections}</div>
+      <div>current: {currentSelections}</div>
+      <div>initialSelections: {initialSelections.map((s) => s.name).join(', ')}</div>
```

Comment:
Debug code?

Reply:
Yup, not sure how i missed removing this ☹️

Reply:
Removing in https://github.com/opendatahub-io/odh-dashboard/pull/3376